src/components/CurriculumComponents/CurriculumDownloadView/index.tsx

```diff
@@ -0,0 +1,249 @@
+import {
+  OakBox,
+  OakFieldError,
+  OakFlex,
+  OakHeading,
+  OakLI,
+  OakP,
+  OakRadioButton,
+  OakRadioGroup,
+  OakUL,
+} from "@oaknational/oak-components";
+import { FC, FormEvent, useState } from "react";
+import styled from "styled-components";
+
+import CurriculumDocumentPreview from "../CurriculumDocumentPreview";
+import AcceptTerms from "../OakComponentsKitchen/AcceptTerms";
+import YourDetails from "../OakComponentsKitchen/YourDetails";
```

Comment:
Just as a sidenote, I think the useResourceFormState in TeacherComponents is a hook for managing downloads form state which might be useful